### ISSUE TITLE
Permission denied error after customization of image is complete.

### ISSUE THREAD

**Issue by hmaarrfk** [PJ-Singh-001/Cubic#30]
**Describe the bug**

I'm trying to customize an image.

I generally:
1. Disable `Update Release Description`
```
add-apt-repository universe
apt update
apt install --yes vim tmate tmux openssh-server git curl wget nvme-cli htop mesa-utils gimp chrome-gnome-shell
# Remove the cache from apt to make the image smaller
rm -rf /var/lib/apt/lists/*
```

**Expected behavior**
Cubic to move forward.

I guess some parent directories were destroyed. 
It seems the code in `apt/cache` tries to rebuild the cache, but doesn't have the right permissions.

**OS Information (please complete the following information):**
 - OS/Distro Name: Ubuntu
 - OS Version: Ubuntu 21.10

**Cubic Information (please complete the following information):**
 - Cubic Version: 
 ```
$ dpkg -l cubic
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version                                       Architecture Description
+++-==============-=============================================-============-====================================================
ii  cubic          2022.01-70-release~202201240318~ubuntu21.10.1 all          Wizard to create a customized Ubuntu Live ISO image.
```
 - ISO Customizing: `ubuntu-20.04.4-desktop-amd64.iso`

**Cubic Log:**

```
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/share/cubic/cubic/navigator.py", line 595, in navigate
    result = new_page.setup(action, page) if new_page else None
  File "/usr/share/cubic/cubic/pages/prepare_page.py", line 123, in setup
    validate_page()
  File "/usr/share/cubic/cubic/pages/prepare_page.py", line 1483, in validate_page
    model.ubiquity_version = constructor.get_package_version('ubiquity', model.project.custom_root_directory)
  File "/usr/share/cubic/cubic/utilities/constructor.py", line 376, in get_package_version
    apt_cache = apt.Cache(rootdir=root_directory)
  File "/usr/lib/python3/dist-packages/apt/cache.py", line 143, in __init__
    self._check_and_create_required_dirs(rootdir)
  File "/usr/lib/python3/dist-packages/apt/cache.py", line 183, in _check_and_create_required_dirs
    os.makedirs(rootdir + d)
  File "/usr/lib/python3.9/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/mark/Downloads/ubuntu_focal/custom-root/var/lib/apt/lists/partial'
```

**Screenshots**

![image](https://user-images.githubusercontent.com/90008/157067460-d1decc12-9e26-4ef5-8833-0a245bf5335f.png)

**Comment by hmaarrfk** [comment]
I can confirm that adding:
```
mkdir -p /var/lib/apt/lists/partial
```

To my customization routine, helps the system move forward.

I guess writing bug reports helps as much with the debugging process.

I have a workaround for myself, so hopefully this helps others get around the issue too.

**Comment by PJ-Singh-001** [comment]
Thanks for the bug report and for sharing a work-around!

**Comment by hmaarrfk** [comment]
Thanks for making useful software

**Comment by garciassolutions** [comment]
> I can confirm that adding:
> 
> ```
> mkdir -p /var/lib/apt/lists/partial
> ```
> 
> To my customization routine, helps the system move forward.
> 
> I guess writing bug reports helps as much with the debugging process.
> 
> I have a workaround for myself, so hopefully this helps others get around the issue too.

You did this at the end of your customization script?
I am running into the same issue, and when setting the permissions manually throws another error.
Attempting to add `mkdir -p /var/lib/apt/lists/partial` to the end of my install script had no effect.


Update: Only when not removing `/var/lib/apt/lists/` was i able to build my image properly.

**Comment by PJ-Singh-001** [comment]
hmaarrfk, garciassolutions,

You are both removing all files under `/var/lib/apt/lists/` using the following command?

    rm -rf /var/lib/apt/lists/*

After you execute the above, try executing...

    apt update

...and see if this resolves the issue. (You must do this _before_ leaving Cubic's Terminal page).

**Comment by PJ-Singh-001** [comment]
For future reference, this issue is similar to [Cubic does not open in Linux Mint 20.3](https://github.com/PJ-Singh-001/Cubic/issues/65).

**Comment by hmaarrfk** [comment]
i haven't had time to try your suggestion. I'm trying to avoid calling `apt update`. So creating the directories manually seems to be better given those constraints....

**Comment by PJ-Singh-001** [comment]
In progress.

The fix is targeted for Cubic version 2022.01.71.

**Comment by PJ-Singh-001** [comment]
Fix released in Cubic version 2022.01.71.
